examples/panel-basic/src/components/SimplePanel/SimplePanel.tsx

```diff
@@ -4,10 +4,10 @@ import { TimeSeries, TooltipPlugin, TooltipDisplayMode, ZoomPlugin } from '@graf
 import { SimpleOptions } from '../../types';
 import { testIds } from '../testIds';
 
-interface Props extends PanelProps<SimpleOptions> {}
+interface Props extends PanelProps<SimpleOptions> {} /** Entry point */
```

Comment:
For someone reading the code I believe this kind of comment just becomes noise. So I would vote for skipping simple comments as this.

```suggestion
interface Props extends PanelProps<SimpleOptions> {}
```

Reply:
That sounds about right to me. I've removed "Entry point" comment. If there are other specific comments you believe are noise, let me know and I'll remove them as well.